File: Python/NumberSystem/Palindrom.py
def is_palindrome(num):
    temp = num
    rev = 0
    while num > 0:  #23
        dig = num % 10 
        rev = rev * 10 + dig # -> 230+2
        num = num // 10  
    return temp == rev

num = int(input("Enter a number: "))
if is_palindrome(num):   
    print("This number is a palindrome number!")
else:  
    print("This number is not a palindrome number!")

File: Java/NumberSystem/Palindrom.java
import java.util.Scanner;

public class Palindrom{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        int temp = num; 
        int rev = 0;
        while (num>0) {
            int digit = num % 10;
            rev = rev*10 +digit;
            num = num /10;
        }
        if(temp == rev){
            System.out.println("The number is a palindrom number!");
        }
        else{
            System.out.println("The number is not a palindrom!");
        }
    }
}